package.json

```diff
@@ -75,6 +75,7 @@
     "d3-time-format": "^2.2.3",
     "delaunay-triangulate": "^1.1.6",
     "es6-promise": "^4.2.8",
+    "extra-iterable": "^2.5.12",
```

Comment:
This module shouldn't end up in our dependencies; whereas it is only used in new tests. 
To move it from `dependencies` to `dev-dependencies` you could run the commands below which would install the latest version `2.5.13`:
```
npm uninstall extra-iterable
npm install --save-dev extra-iterable
```